agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaEventServiceImpl.java

```diff
@@ -751,14 +751,14 @@ public Event deleteEventById(long eventId, long userIdentityId) throws IllegalAc
       throw new IllegalAccessException("User " + userIdentityId + " hasn't enough privileges to delete event with id " + eventId);
     }
     EventAttendeeList eventAttendeeList = attendeeService.getEventAttendees(event.getId());
-
-    agendaEventStorage.deleteEventById(eventId);
-
     event.setModifierId(userIdentityId);
 
     AgendaEventModification eventModifications = new AgendaEventModification(eventId, event.getCalendarId(), userIdentityId);
     eventModifications.addModificationType(AgendaEventModificationType.DELETED);
     attendeeService.sendInvitations(event, eventAttendeeList.getEventAttendees(), eventModifications);
+
+    agendaEventStorage.deleteEventById(eventId);
+
```

Comment:
Delete before sending notifications

```suggestion
    agendaEventStorage.deleteEventById(eventId);
    attendeeService.sendInvitations(event, eventAttendeeList.getEventAttendees(), eventModifications);
```